docs/ig/index.html

```diff
@@ -8,14 +8,14 @@ <h2>W3C WoT – Interest Group</h2>
 	<p>
 		The W3C WoT Interest Group (IG) plays a complementary role to the Working Group. The IG organizes and runs PlugFests to evaluate the current working assumptions, reaches out and collaborates with interested organizations, vendors, and communities, and explores new areas to identify work that is ready for transfer to the W3C Recommendation Track (i.e., any W3C Working Group).
 	</p>
-	<p>Co-Chairs: <a href="mailto:michael.mccool@intel.com">Michael McCool (Intel Corp.)</a> and <a href="mailto:sebastian.kaebisch@siemens.com">Sebastian K&auml;bisch (Siemens AG)</a></p>
-	<p>Team contacts: <a href="https://www.w3.org/People#ashimura">Kazuyuki Ashimura</a> and <a href="https://www.w3.org/People#dsr">Dave Raggett</a></p>
+	<p>Co-Chairs: <a href="mailto:michael.mccool@intel.com">Michael McCool (Intel Corp.)</a>, <a href="mailto:sebastian.kaebisch@siemens.com">Sebastian K&auml;bisch (Siemens AG)</a> and <a href="mailto:michaeljohnkoster@gmail.com">Michael Koster</a></p>
```

Comment:
```suggestion
	<p>Co-Chairs: <a href="mailto:michael.mccool@intel.com">Michael McCool (Intel Corp.)</a>, <a href="mailto:sebastian.kaebisch@siemens.com">Sebastian K&auml;bisch (Siemens AG)</a> and <a href="mailto:michaeljohnkoster@gmail.com">Michael Koster (Invited Expert)</a></p>
```

Minor: Do we want to match what the charter itself says?

Reply:
Ah, sorry. It's my mis-copy, and I've just fixed it. Thanks a lot, @danielpeintner !